examples/packages/postgres-metabroker/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+
+: "${IMAGE_TAG:=postgres-metabroker-credential}"
+
+git_root=$(git rev-parse --show-toplevel)
+
+docker buildx build \
+    --tag "${IMAGE_TAG}" \
+    --file "${git_root}/examples/packages/postgres-metabroker/images/Dockerfile.credential" \
+    "${git_root}/examples/packages/postgres-metabroker/images/"
```

Comment:
```suggestion
cd "$(dirname "${0}")"

docker buildx build \
    --tag "${IMAGE_TAG}" \
    --file "Dockerfile.credential" .
```

- There's no need to find the git root; you just want the directory relative to the script file. (This can be useful for CI if we don't have a full checkout.)
- If we `chdir` into the directory first, we don't need the giant command lines.

Reply:
Hmm, yeah, it works too.